lib/ui/table.ts

```diff
@@ -55,17 +60,16 @@ const tableConfigDefaults = {
   },
 };
 
-function getTableContents(tableData = [], tableConfig = {}) {
+export function getTableContents(
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  tableData: any[][] = [],
```

Comment:
I don't think we can. In the `isObject` function, we're explicitly checking whether the `item` argument is an object, so that we can then loop over that object in the `mergeDeep` function. I think this is truly a case where `any` applies.